addons/ness-lambda/cookiecutter.json

```diff
@@ -0,0 +1,27 @@
+{
+    "root": ".",
+    "lambda_name": "sample",
+    "lambda_underscore": "@{{ cookiecutter.lambda_name.lower().replace(' ', '_').replace('-', '_')}}",
+    "description": "What is the lambda doing",
+    "schedule_expression": "cron(0 8 * * ? *)",
+    "schedule_timezone": "Europe/Zurich",
+    "sqs_queue": "sample",
+    "sqs_queue_underscore": "@{{ cookiecutter.sqs_queue.lower().replace(' ', '_').replace('-', '_')}}",
+    "is_triggered_by_scheduler": "true",
```

Comment:
would it make sense to keep just something like:
"scheduler" and offering the possibility to chose between sqs and scheduler? If I get it right they would exclude each other.

Reply:
You could have both. I prefer the specify each so we could eventually add a 3th way to trigger the lambda.

Reply:
Ok, if we can have both, it makes sense. Still we could simplify a bit by using `scheduler_trigger` and `sqs_trigger` names. The prompt will anyway describe the field. But it's more a cometic improvement.